clusterloader2/testing/l4lb/config.yaml

```diff
@@ -0,0 +1,95 @@
+# LOAD_BALANCER_BACKEND_SIZE specifies the number of backend pods behind each LB
+{{$LOAD_BALANCER_BACKEND_SIZE := DefaultParam .CL2_LOAD_BALANCER_BACKEND_SIZE 5}}
+# LOAD_BALANCER_REPLICAS specifies the number of Load balancer type service
+{{$LOAD_BALANCER_REPLICAS := DefaultParam .CL2_LOAD_BALANCER_REPLICAS 3}}
+# LOAD_BALANCER_TYPE specifies the type of L4 LB created. Valid values are "INTERNAL" and "EXTERNAL"
+{{$LOAD_BALANCER_TYPE := DefaultParam .CL2_LOAD_BALANCER_TYPE "EXTERNAL"}}
+# $EXTERNAL_TRAFFIC_POLICY specifies the externalTrafficPolicy on LB type service. Valid values are "Cluster" and "Local"
+{{$EXTERNAL_TRAFFIC_POLICY := DefaultParam .CL2_EXTERNAL_TRAFFIC_POLICY "Cluster"}}
+# $NODE_SYNC_TIMEOUT specifies the timeout to wait for nodesync to complete
+{{$NODE_SYNC_TIMEOUT := DefaultParam .CL2_NODE_SYNC_TIMEOUT "30m"}}
+# adding a fixed value for first version of the test, rate of pod creation not a concern yet.
+{{$lbQPS := 500}}
```

Comment:
The biggest clusters can handle 100 qps pod creation and 1/3 of that for deletion. Default controller-manager qps limit (for smaller clusters) is 20 (deletion also 1/3 of that)

Reply:
ack